pkg/controller/mongodb/replica_set_controller.go

```diff
@@ -452,7 +455,14 @@ func (r ReplicaSetReconciler) buildAutomationConfigConfigMap(mdb mdbv1.MongoDB)
 		return corev1.ConfigMap{}, err
 	}
 
-	ac, err := buildAutomationConfig(mdb, manifest.BuildsForVersion(mdb.Spec.Version), currentAC, authModification, tlsModification)
+	ac, err := buildAutomationConfig(
+		mdb,
+		manifest.BuildsForVersion(mdb.Spec.Version),
+		currentAC,
+		authModification,
+		tlsModification,
+		getMongodConfigModification(mdb),
```

Comment:
`buildAutomationConfig` already has `mdb` as an argument, do we need to pass `getMongodConfigModification(mdb)` as an additional one? Maybe we can call `getMongodConfigModification(mdb)` from inside `buildAutomationConfig` what do you think?

Reply:
Good idea! You think this should be a new field on the builder or should we keep up the use of `Modification`?

Reply:
I think we can do a Modification!